apps/src/templates/sectionProgress/progressTables/ProgressTableDetailCell.story.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import ProgressTableDetailCell from './ProgressTableDetailCell';
+import {
+  fakeLevels,
+  fakeLevel,
+  fakeProgressForLevels
+} from '../../progress/progressTestHelpers';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+
+const statusForLevel = [
+  LevelStatus.perfect,
+  LevelStatus.not_tried,
+  LevelStatus.attempted,
+  LevelStatus.passed,
+  LevelStatus.submitted
+];
+const levels = fakeLevels(5);
+levels[0].isConceptLevel = true;
+
+const diamondLevels = fakeLevels(2);
+diamondLevels[0].isConceptLevel = true;
+diamondLevels[1].isConceptLevel = true;
+
+const unpluggedLevel = fakeLevel({id: 2, isUnplugged: true});
+
+const studentProgress = fakeProgressForLevels(levels);
+levels.forEach(
+  (level, index) => (studentProgress[level.id].status = statusForLevel[index])
+);
+
+const pairedProgress = fakeProgressForLevels(levels);
+pairedProgress[levels[0].id].paired = true;
+
+const sublevels = fakeLevels(5);
+sublevels.forEach((sub, index) => {
+  sub.bubbleText = String.fromCharCode('a'.charCodeAt(0) + index);
+});
+const levelWithSublevels = fakeLevels(1)[0];
+levelWithSublevels.sublevels = sublevels;
+
+function wrapped(component) {
```

Comment:
A comment here about why we need to wrap the component might be helpful.

Reply:
👍